src/components/address-search/AddressSearchInput.js

```diff
@@ -308,14 +245,14 @@ class AddressSearchInput extends Component<Props, State> {
 
     const fetchResults = async() => {
       const fiResponse = await fetchByKeyword('fi');
-      const svResponse = await fetchByKeyword('sv');
+      // const svResponse = await fetchByKeyword('sv');
 
       const fiResults = await fiResponse.json();
-      const svResults = await svResponse.json();
+      // const svResults = await svResponse.json();
 
       return [
-        ...fiResults.results.map((street) => ({...street, language: 'fi'})),
-        ...svResults.results.map((street) => ({...street, language: 'sv'})),
+        ...fiResults.results.map((address) => ({...address, language: 'fi'})),
+        // ...svResults.results.map((address) => ({...address, language: 'sv'})),
```

Comment:
Why having code for Swedish results, but not use them? Should consider removing this from the codebase.

Reply:
Removed